lib/project.ts

```diff
@@ -164,7 +164,10 @@ export class Project implements Project.IProject {
 				return !isExcluded && !isSubprojectDir;
 			});
 
-			var ignoreFilesRules = this.$pathFilteringService.getRulesFromFile(path.join(projectDir, Project.IGNORE_FILE));
+			var ignoreFilesRules = <string[]>_(this.ignoreFilesConfigurations)
```

Comment:
Is this cast required? Can we use:
```Typescript
var ignoreFilesRules: string[] = _(this.ignoreFilesConfigurations)
```
(looks like I really don't like <...> cast syntax )